domain/repositories/champion.go

```diff
@@ -0,0 +1,121 @@
+package repositories
+
+import (
+	"database/sql"
+	"errors"
+
+	"github.com/wizelineacademy/golang-bootcamp-2020/domain/models"
+)
+
+// Errors
+
+// ErrNoRecord is a custom error used in case no Champion is found in the database
+var ErrNoRecord = errors.New("repositories: no matching record found")
+
+// ChampionRepository defines the interface used by a Champion struct to access its repositories methods
+type ChampionRepository interface {
+	GetSingle(id int) (*models.Champion, error)
+	GetMultiple(limit int) ([]*models.Champion, error)
+	Insert(champion *models.Champion) (int, error)
+}
+
+// ChampRepo defines the link between the Champion and the Database
+type ChampRepo struct {
+	DB *sql.DB
+}
+
+// NewChampRepo returns an inituialized ChampRepo struct
+func NewChampRepo(DB *sql.DB) *ChampRepo {
+	return &ChampRepo{DB}
+}
```

Comment:
By convection, `DB` should be changed to `db`.
```suggestion
func NewChampRepo(db *sql.DB) *ChampRepo {
	return &ChampRepo{db}
}
```